Components/GridListSteps/_style.scss

```diff
@@ -1,6 +1,17 @@
 [is='flynt-grid-list-steps'] {
+  $step-list-spacing: 20px;
+  $step-list-max-width: 380px;
+
   background-color: rgba(0, 0, 0, 0.03);
-  padding: $component-spacing-desktop 0;
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    padding: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
+  }
```

Comment:
Could you use the class `.centerMaxWidthContainer` on the `.container` element instead? (like used in the BlockWysiwygSidebar component)

Reply:
Will use the class as suggested.
Please note that `.centerMaxWidthContainer` class does not contain`$component-spacing-mobile` variable

Reply:
Class `.centerMaxWidthContainer` is dealing with the horizontal spacing. 
Styles marked here are defining the vertical spacing of the components.

I will add `.centerMaxWidthContainer` class in the markup.

Reply:
ah sure, understand.